1.table.sql

```diff
@@ -0,0 +1,42 @@
+use CostumeDB
+go
+drop table if exists Sales
+go
+
+create table dbo.Sales(
+    CustomerId int not null identity primary key,
+    CustomerName varchar(50) not null 
```

Comment:
AS: It did not ask for it specifically but split up the First and Last name.

Reply:
Done